envs/env.py

```diff
@@ -347,8 +356,7 @@ def _start_external_processes(self, reset_time=True):
         logging.info("All commands executed. Episode started!")
 
     def set_random_variation_step(self):
-        random_step = int(self.max_time_steps_per_episode/3) if self._is_testing else np.random.randint(self.max_time_steps_per_episode/4, 3*self.max_time_steps_per_episode/4)
-        return random_step
+        return self.random_variation_step
```

Comment:
I think we don't need this method anymore, as we are just returning `self.random_variation_step` and the method is a class method.